### PR TITLE
Feature/copyright logo override

### PR DESCRIPTION
**Pull request by oestrogen**
Logo is now to the right and overridable:

SCR-20230921-kyzt

### CODE REVIEW

File: docs/override.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ or changed, features.
...
+You can add the logotype of your organization to the footer by pointing out your own .svg file.
+
+
+**Important! It has to be .svg**
+```yaml
+bankid:
+  ...
+  ui: 
+    override:
+      svgLogo: "path/to/my/logo.svg"
```
Comment (martin-lindstrom) [inline docs/override.md:29]: Use `svg-logo`. We have that convention.

File: docs/override.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ or changed, features.
 <a name="customizing-the-bankid-idp-ui"></a>
 ## Customizing the BankID IdP UI
 
+### Adding your logotype
+
+The header will always display the relaying party logotype. 
+You can add the logotype of your organization to the footer by pointing out your own .svg file.
+
+
+**Important! It has to be .svg**
```
Comment (martin-lindstrom) [inline docs/override.md:23]: Re-phrase:

Note that the logotype must in SVG-format.

File: docs/override.md [hunk 3]
```diff
@@ -14,6 +14,20 @@ or changed, features.
 <a name="customizing-the-bankid-idp-ui"></a>
 ## Customizing the BankID IdP UI
 
+### Adding your logotype
```
Comment (martin-lindstrom) [inline docs/override.md:17]: We use Camel-case for headings:

"Adding Your Logotype"

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Need documentation in configuration.md - https://docs.swedenconnect.se/bankid-saml-idp/configuration.html#ui-configuration.

Also some comments ...

**Review by felix-hellman — APPROVED** [review]

**Review by felix-hellman — APPROVED** [review]

**Pull request merged**